conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,265 @@
+* **Buttons**
+  * **Next and back buttons**: Buttons for navigating the steps and modules within each task of a quick start
+  * **Final screen buttons**: Buttons for closing the quick start, going back to previous tasks within the quick start, and viewing all quick starts
+
+
+## <a name="partii"></a> Part II: How does a quick start work?
+This is the typical flow a user follows to complete a quick start:
+1. In the Administrator or Developer perspective, click the **Help** icon and select **Quick starts**.
+
+<img src="../images/qs-help-menu.png" alt="help menu with Quick starts item" width="450"/>
+
+2. Click a quick start card.
+
+<img src="../images/qs-click-card.png" alt="Quick starts card example" width="450"/>
+
+3. In the panel that appears, click **Start**.
+
+<img src="../images/qs-introduction.png" alt="Quick starts introduction screen with description, task headings, and Start button" width="450"/>
+
+4. Complete the on-screen instructions.
+5. Click **Next**.
+6. In the **Check your work** module that appears, answer the question to confirm if you've successfully completed the task.
+  * If you select **Yes**, click **Next** to continue to the next task.
+  * If you select **No**, repeat the task instructions and check your work again.
+
+<img src="../images/qs-check-your-work.png" alt="check your work module" width="450"/>
+
+7. Repeat steps 1-6 above to complete the remaining tasks in the quick start.
+8. After completing the final task, click **Close** to close the quick start.
+
+<img src="../images/qs-last-screen.png" alt="last screen with completed task headings, Close button, and Back button" width="450"/>
+
+
+## <a name="partiii"></a> Part III: How do I write a quick start?
+**Follow these existing style resources for quick start content:**
+* For technical content stylistic requirements, refer to the [CCS product documentation guide](https://redhat-documentation.github.io/supplementary-style-guide/).
+* For product naming, refer to the [Red Hat Official Product Naming List](https://docs.google.com/spreadsheets/u/2/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/pubhtml?gid=1259317633&single=true).
+* For voice and tone requirements, refer to [PatternFly’s brand voice and tone guidelines](https://www.patternfly.org/v4/ux-writing/brand-voice-and-tone).
+    * For other UX content guidance, refer to all areas of [PatternFly’s UX writing style guide](https://www.patternfly.org/v4/ux-writing/about).
+    * For any UX content guidance that’s not covered in PatternFly, refer to [IBM Carbon’s UX content guidelines](https://www.carbondesignsystem.com/guidelines/content/guidance).
+
+The following guidelines provide some more information specific to quick starts, but be sure to refer to the style resources linked above when writing quick start content.
+
+**Main content areas of a quick start:**
+* Card copy
+* Introduction
+* Task steps
+* Modals and in-app messaging
+* Check your work module
+* Completion confirmation
```

Comment:
We should remove "Completion confirmation" since those won't be included moving forward.